packages/card/src/Card.ts

```diff
@@ -112,8 +123,17 @@ export class Card extends ObserveSlotPresence(
 
     private handleKeydown(event: KeyboardEvent): void {
         const { code } = event;
-        if (code === 'Space') {
-            this.toggleSelected();
+        switch (code) {
+            case 'Space':
+                this.toggleSelected();
+                if (this.toggles) {
+                    break;
+                }
+            case 'Enter':
+            case 'NumpadEnter':
+                if (this.href) {
+                    this.likeAnchor?.click();
```

Comment:
Is this meant to be `this.click()`?

If so, should `this.click` begin with `if (!this.href) return`?

Reply:
In this case, we're specifically _not_ clicking `this`. We could, I guess, but we'd still want to gate it by `this.href` as these keys only map to a `click` interaction when `href` is present.

Reply:
Isn't clicking `this` and clicking `this.likeAnchor` the same thing, due to overriding `this.click`? ie:

https://github.com/adobe/spectrum-web-components/blob/2c491258febe57aa7c94827f243a64f81f465cd4/packages/card/src/Card.ts#L102

Just making sure I really grok what's going on.